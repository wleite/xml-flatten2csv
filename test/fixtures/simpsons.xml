<Simpsons>
    <Episode>
        <Title>Cape Feare</Title>
        <People>
            <Person>
                <Name>Maggie</Name>
                <Age>3</Age>
                <Siblings>
                    <Brother>Bart</Brother>
                    <Sister>Lisa</Sister>
                </Siblings>
            </Person>
            <Person>
                <Name>Lisa</Name>
                <Age>3</Age>
                <Siblings>
                    <Brother>Bart</Brother>
                    <Sister>Maggie</Sister>
                </Siblings>
            </Person>
            <Person>
                <Name>Marge</Name>
                <Age>36</Age>
            </Person>
            <Person>
                <Name>Sideshow Bob</Name>
            </Person>
        </People>
    </Episode>
    <Episode>
        <Title>Homer Loves Flanders</Title>
        <People>
            <Person>
                <Name>Homer</Name>
                <Age>39</Age>
            </Person>
            <Person>
                <Name>Ned Flanders</Name>
                <Age>60</Age>
            </Person>
        </People>
    </Episode>
</Simpsons>
