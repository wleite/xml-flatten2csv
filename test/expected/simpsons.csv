"title","name","age","sister","brother"
"Cape Feare","Maggie",1,"Lisa","Bart"
"Cape Feare","Lisa",8,"Maggie","Bart"
"Cape Feare","Bart",10,"Lisa","Maggie (wait! what?)"
"Cape Feare","Marge",36,,
"Cape Feare","Sideshow Bob",,,
"Homer Loves Flanders","Homer",39,,
"Homer Loves Flanders","Ned Flanders",60,"Nediana",
